### ISSUE TITLE
How to add splashscreen

### ISSUE THREAD

**Issue by giksa** [mwaylabs/generator-m-ionic#457]
The guide (readme) isn't clear about how i can add the splashscreen images and the icons.
With ionic i can run the command ionic resources, but with this generator i don't know how.

Can someone explain the steps to add splashscreen images and icon images?

**Comment by millerf** [comment]
Everything is written up here. It has example and much more.
https://github.com/mwaylabs/generator-m-ionic/blob/master/docs/guides/icons_splash_screens.md

It cannot be more clear. But if you have a problem, please explain more in details what steps you followed and what doesn't work.

**Comment by giksa** [comment]
Well, the docs aren't really clear.
The Docs of IONIC are clear:
http://ionicframework.com/docs/cli/icon-splashscreen.html
They say where to put the images, what i have to edit, why i have to do that, how to resize the images, etc.
In this generator i saw that i have to put the icon in the res/*/(default|current|etc) and link it in the config.xml file, but.. what about the splashscreen and the different sizes of it? I have to link every file of the splashscreen manually? (this doesn't seems to be more easy that default ionic tool, i mean, this generator is about to make our development life more easier isn't?)

**Comment by millerf** [comment]
Ok, now I understand you a bit more :)
Yes, so far you need to create the different sizes of images, and link each of them in your config file.
The generator "only" allows you to provide different SETS of images without having to change your config file.

You might not find that it is enough, but the splashcreens and icons are not usually a MASSIVE part of the developmment of an app :) Once they are set up, you usually donn't change them.
On another note, it would be quite time consuming to automatically downscale/upscale all images (as I think that was your idea) at each build.

**Comment by giksa** [comment]
Yes, that i what i want. I agree that isn't a massive part of the development, but it's an important part for the client. I mean, the interface and this details i what the final user and the client see, and sometimes they want to change that and well, it's a very boring task.
i don't think about it like a task that must be executed in the watch task, i mean, it must be a single command that has to do that.

**Comment by millerf** [comment]
I can suggest that you use those kid of tools to generate your images:

http://ticons.fokkezb.nl/

It even has a CLI module,...

**Comment by MathiasTim** [comment]
hey you. I fully agree @millerf, just so you understand it a little bit better: The generator is also for "pros". Which means in our case we have a separated department for design and "generating" splash screens is no option for them. Thats, because we often use very unique and custom splash screens (especially on iOS).
Thats why we have less "magic" than the ionic cli has, because its a process that is setup once and we just replace the images sometimes.

P.S.: Our design department uses [a sketch extension](https://github.com/felvct/sketch-mobile-assets) to help with the different sizes and stuff
